cmd/geth/chaincmd.go

```diff
@@ -341,6 +341,9 @@ func exportChain(ctx *cli.Context) error {
 		if first < 0 || last < 0 {
 			utils.Fatalf("Export error: block number must be greater than 0\n")
 		}
+		if head := chain.CurrentBlock(); uint64(last) > head.NumberU64() {
```

Comment:
If you were to do CurrentFastBlock, then export would work on a not-yet-synced node too like now.

Reply:
Yup, ping on this -- the exporter doesn't need the full CurrentBlock, it's sufficient with the CurrentFastBlock, which has the full body of the block.

Reply:
Ah yes that's better. Updated to use `CurrentFastBlock`. If the PR is okay otherwise I can squash the commits